### PR TITLE
fix: close swipeable list item on any click action

### PR DESCRIPTION
**Pull request by nada-deriv**

### CODE REVIEW

File: src/components/data-table.jsx [hunk 1]
```diff
@@ -33,15 +33,17 @@ const DataTable = (props) => {
             {(cell_value, index) => (
               <tr
                 draggable="true"
-                onTouchStart={() => setActiveIndex(index())}
+                onTouchStart={() =>
+                  setActiveIndex(`${index()}-${cell_value.tick}`)
```
Comment (farrah-deriv) [inline src/components/data-table.jsx]:
```suggestion
                  setActiveIndex(index() - cell_value.tick)
```

Comment (nada-deriv) [reply src/components/data-table.jsx]: Since index() is a num and cell_value.tick is a string, the resultant will be NaN, which then wont work

Comment (likhith-deriv) [reply src/components/data-table.jsx]: Nit: I think you can  use only cell_value.tick, no need to use combination of the index & the symbol.

Comment (nada-deriv) [reply src/components/data-table.jsx]: updated

**Review by likhith-deriv — APPROVED** [review]

**Review by farrah-deriv — APPROVED** [review]

**Pull request merged**